.github/workflows/lint-haml.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches-ignore:
       - 'dependabot/**'
```

Comment:
These lines can be dropped

Reply:
I wanted to leave them here, as there might be dependabot branches still open, or it might get re-enabled somehow one day.